docs/api.apib

```diff
@@ -141,21 +226,47 @@ Creates a new image based on a JSON object containing the following metadata.
 + Parameters
     + image_id: `51b39543-9de1-4751-9fe2-48c8d6038ba1` (guid) - Unique ID
     
-    
++ Request
+    [Image Binaries][]
+
++ Response 200 (application/json)
+
+
+### Posting an binary to a non-existing [POST]
```

Comment:
> Posting **a** binary to a non-existing *image?*

As earlier, this might be best done using nested request/response pairs on a single POST stanza

Reply:
Unfortunately, this is a limitation of API blueprint (or at least for both aglio and dredd) that each stanza can have only one set of parameters. So, for this last case I will leave them separate